pkg/eventstometrics/eventstometrics_integration_test.go

```diff
@@ -0,0 +1,109 @@
+// +build integration
+
+package eventstometrics
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	nr "github.com/newrelic/newrelic-client-go/pkg/testhelpers"
+)
+
+var (
```

Comment:
Thoughts on moving this `var` declaration, scoping it to within the `TestIntegrationEventsToMetrics` test 
function?

Reply:
👍